packages/ecc-client-ga4gh-tes/README.md

```diff
@@ -0,0 +1,108 @@
+# ecc-client-ga4gh-tes Components
+
+Web componets for TES
+
+## Introduction
+This package provides a collection of Web Components for interacting with TES (Task Execution System). It offers functionalities for listing, creating, and deleting task runs. These components facilitate seamless integration and enhanced user experience when working with TES tasks.
+### Components
+| Component              | Issue  | Status             |
+|------------------------|--------|--------------------|
+|                        |        |                    |
```

Comment:
Please explain since there are column of issue and status, Is this how I am expected to fill?

| Component              | Issue  | Status             |
|------------------------|--------|--------------------|
|                  ecc-tes-get-run      |     #48   | Done                   |
| ecc-tes-get-runs | #48 | In Progress|

Reply:
Yes! LGTM!